docs/gds-supported-platform/making-things-private-in-harbor.md

```diff
@@ -44,7 +44,7 @@ private.
 The crucial line is `public: "false"` to ensure that anonymous users (ie
 public users) can't pull images from that Harbor repo.
 
-2. Add `imagePullSecrets` to your Helm chart in order to use the
+2. Add `imagePullSecrets` to your Kubernetes deployment-files (pods) in order to use the
```

Comment:
@issyl0 is the code below what you need to add to your Kubernetes deployment-files (pods)?

Reply:
The code below is definitely from a `pod` rather than a `chart`. A Helm chart is a collection of files that together define and render Kubernetes resources. 

Baring in mind that the Kubernetes resource in reference has an `image` tag, it would be reasonable that it's a pod.

But I'll let @issyl0 rule on that. :)

Reply:
As long as it's clear to the user, that's all good. I just wanted to check as I'm not an SME for this :-)

Reply:
Fair. 

I've been going through the documentation today, as we are performing these steps make our pipeline push to Harbor, and it caught me off-guard when I saw a reference to Charts. 

That being said, it's an easy distinction to miss. That's why I hoped this might make it a bit clearer. :)

Reply:
Yep.